.gitignore

```
@@ -43,3 +43,4 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+.vscode/launch.json
```

I suggest ignoring the whole `.vscode` folder.
```suggestion
.vscode/*
```